src/main/cpp/subsystems/Claw.cpp

```diff
@@ -0,0 +1,27 @@
+#include "subsystems/Claw.h"
+
+bool Claw::Init(){
+    return true;
+}
+
+bool Claw::ResetPositionEncoder(){
+    return true;
+}
+bool Claw::ResetPositionPID(){
+    return true;
+}
+bool Claw::SetPosition(){
+    // SetPosition (30); //cone
+    // SetPosition (0); //close
+    // SetPosition (50);//cube
```

Comment:
I mean, really this PR has no functionality, but at least getting the framework is something